x-pack/test/functional/apps/endpoint/search_bar.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default ({ getPageObjects, getService }: FtrProviderContext) => {
+  const pageObjects = getPageObjects(['common', 'endpoint']);
+
+  describe('Endpoint Search Bar', function() {
```

Comment:
Will this file also cover search bar functionality from other pages within the endpoint app?  We will probably have search in the alerts list page at the very least.  What's the plan to structure tests when we add search bar tests from alerts page or additional tests for the endpoints page?

Reply:
My suggestion would be to move the functionality into a helper file, like we did with Sphinx.